CreditManagement/models.py

```diff
@@ -100,73 +98,310 @@ def __str__(self):
         return "{} | {} | {} → {} | {}".format(self.moment, self.amount, self.source(), self.target(), self.notes)
 
 
-# Todo: this is not yet in use
-class AbstractCreditQuerySet(models.QuerySet):
-    source_column = None
-    target_column = None
```

Comment:
Alles hieronder is niet in gebruik, zal wss ook niet gebruikt worden. Staat nu nog als halve mogenlijkheid.